### DISCUSSION TITLE
Source output not loading

### DISCUSSION THREAD

**Discussion by AlessandroVol23** [vercel/community#3767] [Bug Report]
### Page to Investigate

https://vercel.com/user/project/deployment/source?f=out/500

### Steps to Reproduce

When I try to load the output of a deployment it is stuck on loading. This is the case for all files in different projects.

1. Go to a project
2. Go to a deployment
3. Click on source
4. Click on the tab output
5. Select a file
6. See loading on the right side

https://www.loom.com/share/23b93a1f0bb342e58a0dc032fc8e1bbc

<img width="906" alt="image" src="https://github.com/vercel/community/assets/19362086/2d69e50c-97f5-4a58-a7f6-4823fd59953d">

**Comment by swarnava** [comment]
Hi,

I have forwarded this to relevant team to take a closer look. Thank you!

**Comment by swarnava** [comment]
Hi,

We have fixed the issue.
